service/gcs/core/gcs/gcs.go

```diff
@@ -56,25 +56,23 @@ func NewGCSCore(rtime runtime.Runtime, os oslayer.OS) *gcsCore {
 // containerCacheEntry stores cached information for a single container.
 type containerCacheEntry struct {
 	ID                 string
-	ExitStatus         oslayer.ProcessExitState
```

Comment:
I'm fine with that. If we want to have more detailed information we can add it pretty easily later.